airflow/providers/google/cloud/hooks/cloud_build.py

```diff
@@ -55,11 +56,8 @@ class CloudBuildHook(GoogleBaseHook):
     :type impersonation_chain: Union[str, Sequence[str]]
     """
 
-    _conn = None  # type: Optional[Any]
-
     def __init__(
         self,
-        api_version: str = "v1",
```

Comment:
This seems like a not backward-compatible change, can you please add information about all such changes in UPDATING.md?

Reply:
Done.